### PR TITLE
SpongeGradle

### PR DESCRIPTION
**Pull request by kinggoesgaming**
Related #17

### CODE REVIEW

**Comment by Flibio** [comment]
What can't the metafile system be used right now?

**Comment by kinggoesgaming** [comment]
that depends on #11 xD

**Comment by kinggoesgaming** [comment]
The other option I have is to switch to the service instead and use the plugincontainer version

**Comment by Flibio** [comment]
Yes, I think using the PluginContainer version would be better for now. It's much better to only have to change the version once.

**Comment by kinggoesgaming** [comment]
will work on this tomorrow

File: build.gradle [hunk 1]
```diff
@@ -29,13 +28,15 @@ ext {
...
-blossom {
-    replaceToken '@project.version@', project.version, 'src/main/java/me/flibio/updatifier/PluginInfo.java'
-    replaceToken '@project.id@', project.id, 'src/main/java/me/flibio/updatifier/PluginInfo.java'
-    replaceToken '@project.name@', project.name, 'src/main/java/me/flibio/updatifier/PluginInfo.java'
-    replaceToken '@project.description@', project.description, 'src/main/java/me/flibio/updatifier/PluginInfo.java'
+sponge {
+    plugin {
+        id = project.id
+        meta {
+            name = project.name
```
Comment (Minecrell) [inline build.gradle]: This is redundant, that's the default value - same for everything below

Comment (kinggoesgaming) [inline build.gradle]: Ah... I just have templates since the old fully qualified plugin id times

**Comment by kinggoesgaming** [comment]
Decided that this can wait till #11 is discussed and maybe implemented and also target bleeding then

**Pull request closed without merging**